src/org/labkey/test/components/glassLibrary/grids/DetailTableEdit.java

```diff
@@ -121,6 +137,16 @@ public DetailTableEdit setTextField(String fieldCaption, String value)
         return this;
     }
 
+    public DetailTableEdit setInputByFieldName(String fieldName, String value)
```

Comment:
This and the setTextField function feel a little like duplicate functionality. I realize this uses the fieldName and setTextField uses the caption on the form. Maybe at some point in the future the duplication could be removed and a single method is present.

Reply:
There are some fields for which the text in the label or value div just don't work; the special symbol for Extinction Coefficient seems to be a special weird thing that just isn't equal to what you put in code when you search for it in xpath
I honestly would have stuck with the one method, except that the one method just didn't work in some cases